mail_tracking/__manifest__.py

```diff
@@ -7,7 +7,7 @@
 {
     "name": "Email tracking",
     "summary": "Email tracking system for all mails sent",
-    "version": "14.0.2.0.0",
+    "version": "14.0.2.0.1",
```

Comment:
Don't change manually the version. The bot will do it.

Reply:
Done